server/api/products.js

```diff
@@ -48,58 +48,67 @@ router.get('/:productId', (req, res, next) => {
 
 //add new product
 router.post('/', function (req, res, next) {
-  console.log('request body in post is: ', req.body)
-  const { name, description, inventory, price } = req.body;
-  if (!name) {
-    return res.status(400).send('Invalid request; name required');
+  if (req.user && req.user.isAdmin) {
+    const { name, description, inventory, price } = req.body;
+    if (!name) {
+      return res.status(400).send('Invalid request; name required');
```

Comment:
why does the name need to be included? can't it stay the same if admin doesn't want to update it?